d2bs/kolbot/libs/AutoMule.js

```diff
@@ -414,16 +427,52 @@ MainLoop:
 		return true;
 	},
 
+	matchItem: function (item, list) {
+		var i, info, parsedLine,
+			parsedPickit = [], classIDs = [];
+
+		for (i = 0; i < list.length; i += 1) {
+			info = {
+				file: "Character Config",
+				line: list[i]
+			};
+
+			if (typeof list[i] === "number") { // classids
+				classIDs.push(list[i]);
+			} else if (typeof list[i] === "string") { // pickit entries
+				parsedLine = NTIP.ParseLineInt(list[i], info);
+
+				if (parsedLine) {
+					parsedPickit.push(NTIP.ParseLineInt(list[i], info));
+				}
+			}
+		}
+
+		if (classIDs.indexOf(item.classid) > -1 || NTIP.CheckItem(item, parsedPickit)) {
```

Comment:
can combine and just do 
`return (classIDs.indexOf(item.classid) > -1 || NTIP.CheckItem(item, parsedPickit))`

instead of return true / return false